tables/spr.go

```diff
@@ -75,7 +75,7 @@ func (t *SPRTable) Name() string {
 
 func (t *SPRTable) Schema() string {
 
-	sql := `CREATE TABLE %s (
+	sql := `CREATE TABLE %[1]s (
```

Comment:
Oh god, this PR is worth it just for this and not having to `t.Name()` a million times below...